2020/day6/solution.py

```diff
@@ -0,0 +1,44 @@
+import functools
+import operator
+
+class Person:
+
+    @classmethod
+    def from_line(cls, line):
+        person = Person()
+        person.set = set(c for c in line)
```

Comment:
```suggestion
        person.set = set(line)
```
In general comprehensions are also good for non-trivial cases: e.g. `{2*x for x in nums}`